### ISSUE TITLE
Import Launchbox emulated games from a linux version of pegasus 

### ISSUE THREAD

**Issue by jediafr** [mmatyas/pegasus-frontend#798]
Would it be possible to import Launchbox games from a linux version of pegasus ? (many compilations like Exodos, C64forever are created with LB windows but are relying on emulators that could run native to linux, the idea would be to import the LB setting and redirect the emulator towards the linux version through Pegasus). I could provide you with the various /data to improve support.

I have tentatively tried editing the /home/jda/.config/pegasus-frontend/settings.txt like this
providers.launchbox.enabled true (that was a guess)
providers.launchbox.installdir: /run/media/jda/Maxtor/Dock-file/eXoDOS_Lite

But my experiment doesnt work on the latest buld : pegasus-fe_alpha14-112-g539c5cc7_x11-static.zip
Log :
2021-02-16T08:49:41 [w] /home/jda/.config/pegasus-frontend/settings.txt, line 11: line invalid, skipped
2021-02-16T08:49:41 [w] /home/jda/.config/pegasus-frontend/settings.txt, line 12: unrecognized option providers.launchbox.installdir, ignored

**Comment by mmatyas** [comment]
Hi, the LaunchBox support is not built into the Linux release because LaunchBox isn't officially available for Linux either. Other than that, the LaunchBox config files often contain absolute paths, so even if we read the files things like `D:\retroarch\retroarch.exe` won't really make sense in a Linux environment.

**Comment by jediafr** [comment]
I realise it is a very odd request but since you have a working parser for LB i would think that pointing it to the LB directory (regardless of the system) would be possible witjh a minimum of codebase changes. 

Importing those entries in Pegasus metafile would enable us to change the retroarch.exe into the /usr/bin/retroarch... (and probably other paths recquired via a big sed from us).

**Comment by mmatyas** [comment]
Last time I checked LB didn't work with Wine for me, but if there's indeed some intention for that from the authors, that might indeed worth looking into it again. Thanks for the link!

**Comment by jediafr** [comment]
Yeah i it is a bit tricky but i followed instructions in the thread and i can run it (exodos, eXoWin3x and le C64 curated). It is slow as hell but stable. Hence my request as it would speed things greatly and also make use of native emulators, thanks for keep your mind open about this !

**Comment by HVR88** [comment]
Being able to convert exported data from LB for use in Linux/RPi would be a great feature. I run my main EMU system in Windows 10, but I also have a couple of RPi systems I'd like to use Pegasus on - including my on-going Arcade Bar-Top build.

**Comment by polson** [comment]
You could try using this project to export a metafile: https://github.com/spektor56/PegasusExportPlugin
